src/Mapping/Annotation/Slug.php

```diff
@@ -27,12 +27,22 @@ final class Slug implements GedmoAnnotation
 {
     use ForwardCompatibilityTrait;
 
+    public const STYLE_LOWER = 'lower';
+    public const STYLE_UPPER = 'upper';
+    public const STYLE_CAMEL = 'camel';
+    public const STYLE_ORIGINAL = 'default';
```

Comment:
Those styles being a closed list also look like a good candidate for an enum class